Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 104 add a tracking script to frontend

## Related ticket:
[SCRUM-104](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-104?atlOrigin=eyJpIjoiZDYxODljNjdiM2YwNDY0NWJmM2NiY2E2N2E0YmVjYWEiLCJwIjoiaiJ9)

## In this PR:
- Add a Component to inject the tracking script to the end of the body tag

## Testing instruction
- Checkout branch
- Add a Mautic url variable to the .env.local inside Frontend folder: ```VITE_MAUTIC_URL=http://mautic-lando.lndo.site/```
- Refresh the frontend page and the script should be at the end of the Body and working
- The fronted console should not show any error if everything is configured correctly and the cors plugins is enabled

[SCRUM-104]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
